public/docs/_examples/pipes/dart/lib/exponential_strength_pipe.dart

```diff
@@ -0,0 +1,27 @@
+library pipes_examples.exponential_strength_pipe;
+
+import 'package:angular2/angular2.dart';
+import 'dart:math' as math;
+
+/*
+* Raise the value exponentially
+* Takes a value that defaults to 0 and an exponent argument that defaults to 1.
+* Checks for value to be a string or number.
+* Usage:
+*   value | exponentialStrength:exponent
+* Example:
+*   {{ 2 |  exponentialStrength:10}}
+*   formats to: 1024
+*/
+
+@Pipe(name: 'exponentialStrength')
+@Injectable()
+class ExponentialStrengthPipe {
+  transform(dynamic value, [List<dynamic> args]) {
+    var v = int.parse(value.toString(), onError: (source) => 0);
+    var p = args.isEmpty
+        ? 1
+        : int.parse(args[0].toString(), onError: (source) => 1);
```

Comment:
Ok cool. I noticed that when the `args` list is empty that `args[0]` will throw an error ( rather than just return `null` like Javascript would ). Does `List.first` behave the same when empty ?